roaring/roaring.go

```diff
@@ -77,11 +114,16 @@ type Bitmap struct {
 
 // NewBitmap returns a Bitmap with an initial set of values.
 func NewBitmap(a ...uint64) *Bitmap {
-	b := &Bitmap{}
+	b := &Bitmap{
+		Containers: NewSliceContainers(),
+	}
 	b.Add(a...)
 	return b
 }
 
+// NewFileBitmap returns a Bitmap with an initial set of values.
```

Comment:
Why is it `NewFileBitmap`?

Reply:
ok, this comment needs to reflect that NewFileBitmap is specifically for that use case, and may have a different implementation in the enterprise edition (probably mention where that alternate implementation can be found)

Reply:
Yeah sorry, forgot to clarify this in the comment.